### PR TITLE
Add converter for Belgium / wallonia

### PR DESCRIPTION
**Pull request by ivorbosloper**
Data can not be shared by current license

### CODE REVIEW

File: fiboa_cli/convert_utils.py [hunk 1]
```diff
@@ -36,6 +36,9 @@ def convert(
         log("Loading file")
     path = download_file(url, cache_file)
 
+    if not len(bbox) == 4:
+        raise ValueError("Bbox must be of length 4")
+
```
Comment (m-mohr) [inline fiboa_cli/convert_utils.py:41]:
```suggestion
```

I've directly added this to the main branch, good idea.

File: fiboa_cli/datasets/be_wa.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
+from ..convert_utils import convert as convert_
+
+URI = "http://use_your_own_copy/"
```
Comment (m-mohr) [inline fiboa_cli/datasets/be_wa.py:3]: Let me think about a solution that is a bit better than this approach. Maybe we can let users input the path directly when an empty URL is provided through the template. The -c option will change in #48 anyway.

File: fiboa_cli/datasets/be_wa.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    'CULT_NOM': 'crop_name',
+    'GROUPE_CULT': 'group_code',
+}
+
+ADD_COLUMNS = {}
+
+# A list of implemented extension identifiers
+EXTENSIONS = []
+
+import pandas as pd
```
Comment (m-mohr) [inline fiboa_cli/datasets/be_wa.py:43]: Better move the imports to the top of the file.

File: tests/data-files/be_wa.zip
Comment (m-mohr) [inline tests/data-files/be_wa.zip:1]: Can we add this here? It seems the license is very restrictive, which would also apply to a subset of the data...

File: fiboa_cli/convert_utils.py (hunk 1 above)
Comment (ivorbosloper) [reply fiboa_cli/convert_utils.py:41]: Thanks, I forgot it a few times and this error makes it clear from the start.

File: tests/data-files/be_wa.zip
Comment (ivorbosloper) [reply tests/data-files/be_wa.zip:1]: You're right, I thought a small subsample was acceptable. I've now submitted a test file with completely fake data (only using column names and file name).

Comment (m-mohr) [reply tests/data-files/be_wa.zip:1]: Thank you!

**Pull request merged**